filebeat/util/data.go

```diff
@@ -0,0 +1,65 @@
+package util
+
+import (
+	"github.com/elastic/beats/filebeat/input/file"
+	"github.com/elastic/beats/libbeat/common"
+)
+
+type Data struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Data should have comment or be unexported